app/javascript/packs/public.js

```diff
@@ -75,6 +87,32 @@ function main() {
       content.textContent = formattedDate;
     });
 
+    const isToday = date => {
+      const today = new Date();
+
+      return date.getDate() === today.getDate() &&
+        date.getMonth() === today.getMonth() &&
+        date.getFullYear() === today.getFullYear();
+    };
+    const todayFormat = new IntlMessageFormat(messages['relative_format.today'], locale);
```

Comment:
Other direct calls to `IntlMessageFormat` in that file fallback to a hardcoded translation string. I would suggest doing the same.